<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="graphLayout1"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Онлайн измерения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>7</number>
        </property>
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>СТАРТ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>СТОП</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pause_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Пауза</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="graphLayout2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Сохраненные измерения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>7</number>
        </property>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>Сохранить расстояние</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="auto_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Автосохранение</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="cal_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="text">
         <string>Рассчет корекций</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_filter">
     <property name="title">
      <string>Выбор фильтра</string>
     </property>
     <addaction name="actionAverageFilter"/>
     <addaction name="actionExpectationFilter"/>
     <addaction name="actionNoneFilter"/>
    </widget>
    <addaction name="menu_filter"/>
    <addaction name="actionStepSettings"/>
    <addaction name="actionAutoSave"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionAverageFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Среднее арифметическое</string>
   </property>
  </action>
  <action name="actionExpectationFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Фильтр с интерквартильным отклонением</string>
   </property>
  </action>
  <action name="actionStepSettings">
   <property name="text">
    <string>Базовая точка и шаги</string>
   </property>
  </action>
  <action name="actionAutoSave">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Автоматическое сохранение </string>
   </property>
  </action>
  <action name="actionNoneFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Без фильтра</string>
   </property>
  </action>
  <action name="actionBidirectional">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Двунаправленное измерение шагов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
